bulktest/batch_lib.inc

```diff
@@ -964,7 +964,10 @@ function prettyType($mimeType, $ext) {
 	else if ( "xml" === $ext ) {
 		return "xml";
 	}
-	else if ( false !== strpos($mimeType, "flash") || "mp4" === $ext || "swf" === $ext || "f4v" === $ext || "flv" === $ext ) {
+    //Video extensions mp4, webm, ts, m4v, m4s, m4v, mov, ogv
+	else if ( false !== strpos($mimeType, "flash") || (false !== strpos($mimeType, "webm") || (false !== strpos($mimeType, "mp4") || (false !== strpos($mimeType, "flv"))
```

Comment:
I think we can remove the paren group after the flash condition. Also, at the end of this lie we need `||` to join with the next line (fixes a syntax error).